appscale/tools/node_layout.py

```diff
@@ -96,16 +97,23 @@ def __init__(self, options):
         a str containing a path on the local filesystem that, when read,
         contains the YAML in question. It can also be set to None, for
         deployments when the user specifies how many VMs they wish to use.
+    Raises:
+      BadConfigurationException if configuration is not valid.
     """
     if not isinstance(options, dict):
       options = vars(options)
-
+    self.warning = None
     input_yaml = options.get('ips')
     if isinstance(input_yaml, str):
       with open(input_yaml, 'r') as file_handle:
         self.input_yaml = yaml.safe_load(file_handle.read())
     elif isinstance(input_yaml, dict):
       self.input_yaml = input_yaml
+      self.warning = "The AppScalefile is changing, the layout you are using " \
```

Comment:
Yes I wasn't sure if the AppScaleLogger should be imported into NodeLayout, but I guess RemoteHelper does it.